### PR TITLE
E2E/cucumber

### PR DESCRIPTION
**Pull request by ishiasaka**

### CODE REVIEW

File: .github/workflows/build-test-lint.yml [hunk 1]
```diff
@@ -26,9 +64,34 @@ jobs:
...
       - run: pnpm install
-      - run: pnpm run build
       - run: pnpm run lint
       - run: pnpm run test
-      - run: pnpm run test:e2e
         env:
           MONGO_URI: mongodb://localhost:27017/meeorder
+
+  e2e:
+    runs-on: ubuntu-latest]
```
Comment (namirinz) [inline .github/workflows/build-test-lint.yml]: remove ]

File: src/categories/categories.service.ts
Comment (namirinz) [inline src/categories/categories.service.ts:1]: Changing in this file should not be in this PR because it's not relevant with E2E or Cucumber but nvm.

Comment (ishiasaka) [reply src/categories/categories.service.ts:1]: OK,I created pull requested #23 from fix this.

File: .vscode/launch.json [hunk 2]
```diff
@@ -17,24 +17,26 @@
       "env": {
         "NODE_ENV": "development",
         "PORT": "4000",
-        "MONGO_URI": "mongodb://localhost:27017/meeorder"
+        "MONGO_URI": "mongodb://localhost:27017",
+        "MONGODB_NAME": "meeorder"
```
Comment (lazykern) [inline .vscode/launch.json]: According to your code in `features/step-definitions/health.ts`, shouldn't this be `MONGO_DB_NAME`?

File: .vscode/launch.json [hunk 3]
```diff
@@ -17,24 +17,26 @@
...
+      "args": [],
       "console": "integratedTerminal",
       "runtimeVersion": "18",
       "env": {
         "NODE_ENV": "development",
-        "PORT": "4000",
-        "MONGO_URI": "mongodb://localhost:27017/meeorder"
+        "MONGO_URI": "mongodb://localhost:27017/meeorder",
+        "BASE_URL": "http://127.0.0.1:4000",
+        "MONGODB_NAME": "meeorder"
```
Comment (lazykern) [inline .vscode/launch.json]: According to your code in `features/step-definitions/health.ts`, shouldn't this be `MONGO_DB_NAME`?

**Review by lazykern — CHANGES_REQUESTED** [review]

**Review by lazykern — APPROVED** [review]

**Pull request merged**